### ISSUE TITLE
Screen readers don't echo output from the PowerShell Interactive window

### ISSUE THREAD

**Issue by Cambridgeport90** [adamdriscoll/poshprotools#36]
this is the same for both Pro and non-pro versions of this since it's the same feature. When I open a normal PowerShell prompt, and since I use a screen reader to hear any output from the screen as speech, I can tell everything that's going on in the prompt as the output comes down. When running commands in the PowerShell interactive window, the same is not true; I can't hear any output when I press Enter, and I should be able to. This is in the latest Visual Studio, and also the latest version of NVDA. Running a copy of NVDA on your system won't change your experience at all for this, so I'd recommend you run one to see what I'm talking about; that's the best way to describe this.
